client/src/includes/initTagField.ts

```diff
@@ -0,0 +1,40 @@
+import $ from 'jquery';
+
+declare global {
+  interface JQuery {
+    tagit(...args): void;
+  }
+}
+
+/**
+ * Initialises the tag fields using the jQuery tagit widget
+ *
+ *@param id - element id to initialise against
+ *@param source -  auto complete URL source
+ *@param options - Other options passed to jQuery tagit
+ */
+// eslint-disable-next-line @typescript-eslint/ban-types
+const initTagField = (id: string, source: string, options: {}): void => {
```

Comment:
```suggestion
const initTagField = (id: string, source: string, options: Record<string, any>): void => {
```

Try this instead. This is a record (object) with any value.